app/controllers/clearance/passwords_controller.rb

```diff
@@ -88,18 +87,19 @@ def flash_failure_after_update
       default: t('flashes.failure_after_update'))
   end
 
-  def forbid_missing_token
-    if params[:token].to_s.blank?
+  def check_for_forbidden_access
```

Comment:
What do you think about `ensure_token_and_user`?

Currently, none of these actions are really forbidden access so the name feels a bit weird.